Core/Src/torque_manager.c

```diff
@@ -32,9 +31,25 @@ void torque_manager(void *argument) {
 			//TODO: implementar controle longitudinal
 
 			break;
-		case LATERAL:
+		case LATERAL: ;
 			//TODO: implementar controle lateral
 
+			uint32_t tick;
+			tick = osKernelGetTickCount();
+			tick += LATERAL_DELAY;
+			osDelayUntil(tick);
+
+			lateral_t lateral_control();
+			void rampa_torque_lateral(lateral_t *ref_torque_decrease, uint32_t *ref_torque);
+			lateral_t ref_torque_decrease = lateral_control();
+			uint32_t ref_torque_lateral[2] = {0};
+			rampa_torque_lateral(&ref_torque_decrease, ref_torque_lateral); // utilizar rampa_torque enquanto controle longitudinal nao estiver definido
```

Comment:
marca todos esses comentários de coisas a se fazer com um TODO: antes, exemplo:
```suggestion
			rampa_torque_lateral(&ref_torque_decrease, ref_torque_lateral); // TODO: utilizar rampa_torque enquanto controle longitudinal nao estiver definido
```
Isso porque as IDEs destacam TODOs e listam eles num lugar bonitinho, aí fica fácil de saber o q tá pendente sem precisar procurar o código todo.
(muda aqui e em todos os lugares q isso se aplica)